samples/drivers/spi_flash_at45d/sample.yaml

```diff
@@ -0,0 +1,6 @@
+sample:
+  name: SPI Flash AT45D Sample
+tests:
+  sample.drivers.spi.flash.at45d:
+    tags: spi flash
+    platform_whitelist: nrf9160_pca10090
```

Comment:
can harness be added?
This will help to verify sample using sanity-check.
Example of such harness: https://github.com/zephyrproject-rtos/zephyr/blob/master/samples/drivers/soc_flash_nrf/sample.yaml#L7-L23